syncstorage/views/__init__.py

```diff
@@ -179,6 +180,7 @@ def get_lbheartbeat(request):
 
 @info_timestamps.get(accept="application/json", renderer="sync-json",
                      acl=expired_token_acl)
+@check_migration
```

Comment:
This might be a styling nit for me. I'm not a huge fan of decorators to begin with because they secretly alter code flow and/or variables. Burying them inside a meta decorator that may or may not be applied doesn't really help, unless you keep what `default_decorators` does at top of mind at all times. 

That said, we'd need to do this check for any call the client might make which could return a 2xx result. I *could* add it to the `default_decorators` meta, and then add to the additional ones, but since this is kind of a semi-hack specific to the migration, I'd rather be very explicit about what's going on.

Reply:
I'm fine with leaving it separate if that fits best with your mental model 👍